packages/components-angular/package.json

```
@@ -0,0 +1,49 @@
+{
+  "name": "design-system-components-angular-workspace",
```

Always namespace package names for security reasons (dependency confusion). Even if they are not published.

```suggestion
  "name": "@swisspost/design-system-components-angular-workspace",
```